### PR TITLE
Create Temple

### PR DESCRIPTION
**Pull request by kajsaunge**
- [x] Create popup (similar to native android) for datetime picker component ([using display="inline" or so](https://github.com/react-native-datetimepicker/datetimepicker#display-optional))
  - [ ]  Align with design team (might be pushed to next iteration)
- [x] Create temple with selected data
  - [x] Update specs
  - [x] Add translations
- [x]  Locale (uses device settings)
- [x] Fetch real exercise ids for exercise selection

### CODE REVIEW

File: client/src/routes/Temples/Temples.tsx [hunk 1]
```diff
@@ -21,11 +20,13 @@ import NS from '../../lib/i18n/constants/namespaces';
 import {isLoadingAtom, templesAtom} from './state/state';
 import {Temple} from '../../../../shared/src/types/Temple';
 import TempleCard from '../../common/components/Cards/TempleCard/TempleCard';
-import TextInput from '../../common/components/Typography/TextInput/TextInput';
 import SETTINGS from '../../common/constants/settings';
 import {PlusIcon} from '../../common/components/Icons';
 import {GUTTERS, SPACINGS} from '../../common/constants/spacings';
 import {COLORS} from '../../common/constants/colors';
+import {ModalStackProps} from '../../common/constants/routes';
+import {NativeStackNavigationProp} from '@react-navigation/native-stack';
```
Comment (swemail) [inline client/src/routes/Temples/Temples.tsx:28]: Nit pick, but those library import should be at the top

File: client/src/routes/Temples/components/DateTimePicker.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+});
+
+const DoneButton = styled(Button)({alignSelf: 'center'});
+
+const DateTimePicker: React.FC<{
+  mode: 'date' | 'time';
+  selectedValue: dayjs.Dayjs;
+  setValue: Dispatch<SetStateAction<dayjs.Dayjs>>;
+  close: () => void;
+  maximumDate?: Date;
```
Comment (swemail) [inline client/src/routes/Temples/components/DateTimePicker.tsx:60]: Maybe expose these as Dayjs and convert so we hide that the underlying component needs Date

**Review by swemail — APPROVED** [review]
Nice work!! I approve and if you want to consider my suggestions in the comment I will approve again

File: client/src/routes/Temples/components/DateTimePicker.tsx (hunk 2 above)
Comment (Nipher) [reply client/src/routes/Temples/components/DateTimePicker.tsx:60]: Yessss! that's da way for sure, I'll update it! 🔥

**Comment by gewfy** [comment]
I have some ideas on simplifying the content handling. Let’s sync today :)

**Review by gewfy — APPROVED** [review]

**Pull request merged**